### ISSUE TITLE
Open external links on new tab

### ISSUE THREAD

**Issue by vechocho** [gbif/hp-ecuador#2]
We need to figure a way to open external links on a new tab. Is this possible?

**Comment by vechocho** [comment]
I use this extension https://rubygems.org/gems/jekyll-extlinks updating the gemfile adding this:

`gem 'jekyll-extlinks', '~> 0.0.5'`

**Comment by vechocho** [comment]
It didn't work 😢

**Comment by MortenHofft** [comment]
What kind of links is it? In your markdown?

Links generally open in a new tab if you define an attribute `target="_blank"` on the `a` tag.
So in kramdown you could do `[link](url){:target="_blank"}` or you write the html (which is also possible in markdown) and do `<a href="http://example.com/" target="_blank">Hello, world!</a>`

Personally I find it annoying when links are forced into new tabs. If I want to keep the current tab open I see hold down command when clicking. I prefer taking that decision myself. But of course there are exceptions where it makes sense.

**Comment by vechocho** [comment]
Yes i was thinking of links to external pages as I have some features that lin to news on GBIF webpage and other thak link to NEWS on the hosted portal

The kramdown dindt worked for link on "href"

I implemented this one and worked https://rubygems.org/gems/jekyll-target-blank
